src/test/java/se/sundsvall/messaging/integration/db/entity/MessageEntityTest.java

```diff
@@ -0,0 +1,81 @@
+package se.sundsvall.messaging.integration.db.entity;
+
+import static java.time.temporal.ChronoUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.within;
+
+import java.time.LocalDateTime;
+
+import org.junit.jupiter.api.Test;
+
+import se.sundsvall.messaging.model.MessageStatus;
+import se.sundsvall.messaging.model.MessageType;
+import se.sundsvall.messaging.test.annotation.UnitTest;
+
+@UnitTest
+class MessageEntityTest {
+
+	@Test
+	void testBuilder() {
```

Comment:
Could add ```assertThat(bean).hasOnlyFields()``` to ensure that addition of a new field is breaking the test.

Reply:
Not needed, hasNoNullFieldsOrProperties will break if new field is added but not propagated in the test

Reply:
Adding a primitive won't break this test.

Reply:
The testsuite will still break when the DB schema is verified. Having multiple tests doing the same verifications is not needed.